src/Compute/Compute.Test/ScenarioTests/VirtualMachineTests.cs

```diff
@@ -445,7 +445,7 @@ public void TestCapacityReservation()
             TestRunner.RunTestScript("Test-CapacityReservation");
         }
 
-        [Fact]
+        [Fact(Skip = "Cannot run on Github Platform due to ACL API")]
```

Comment:
wait, what happened here???

Reply:
Github tests are run platform independent so it does not support ACL API when generating ssh keys. Tests can be run fine locally.